docs/tutorials/e2e/boost/dtrTutorial.md

```diff
@@ -0,0 +1,431 @@
+
+This tutorial focuses on the working with the DTR. You will learn what the DTR is, how to deploy it and go through a complete use case scenario by depositing a DTR in the EDC, registering a digital twin in the DTR, and showing how external parties can find and consume the digital twin.
+
+## Introduction
+
+Catena-X uses the Asset Administration Shell (AAS) to represent digital twins. Such a digital twin is basically a shell with an ID in UUID-format that makes it uniquely identifiable. Also it contains IDs ("specificAssetIds") that connect the shell with the original asset it represents (e.g. the "manufacturerPartId"). The shell also contains SubmodelDescriptors which reference to submodels that contain the actual data of specific aspects of the asset.
+
+The Digital Twin Registry (DTR) contains a list of all registered digital twins of the owner and therefore acts as an address book for Digital Twins. With the DTR of a data provider, a data consumer can therefore find digital twin and also get directed to the desired submodels of the twin.
+
+Further details can be found in the [Digital Twin KIT][dt-kit] or the [sldt-digital-twin-registry][sldt-dtr] Github repository.
+
+In this tutorial, Bob will assume the role of an automotive supplier that manufactures gearboxes and acts as the data provider. Alice represents an OEM that uses these gearboxes. Alice's goal is to calculate the product carbon footprint (PCF) of a car, and she also needs the carbon footprints of the individual parts to calculate the total footprint. Therefore, Alice is the data consumer in this tutorial.
+
+The steps of the tutorial are as follows
+
+- Bob registers a DTR at his EDC
+- Bob registers a Digital Twin at his DTR
+- Alice finds and consumes the Digital Twin
+
+This tutorial will guide you through the steps outlined above. For a deeper understanding, more detailed documentation will be linked.
+
+### High Level Architecture
+
+## Exchange Digital Twins
+
+### Register a DTR in your EDC
+
+#### Create the DTR asset
+
+To ensure that Bob's DTR becomes visible for Alice and to start the data exchange between them, Bob has to create a data asset.
+
+Action (Bob): Create a data asset using the following command:
+
+(note: that the "asset:prop:type" is standardized with "data.core.digitalTwinRegistry" for the Digital Twin Registry.)
+
+```curl --location 'http://dataprovider-controlplane.tx.test/management/v3/assets' \
+--header 'Content-Type: application/json' \
+--header 'X-Api-Key: TEST2' \
+--data-raw '{
+   "@context":{
+      "edc":"https://w3id.org/edc/v0.0.1/ns/",
+      "cx-common":"https://w3id.org/catenax/ontology/common#",
+      "cx-taxo":"https://w3id.org/catenax/taxonomy#",
+      "dct":"https://purl.org/dc/terms/"
```

Comment:
```suggestion
      "dct":"http://purl.org/dc/terms/"
```
This will come up multiple times, there used to be an error in a spec somewhere that spread. For reference, check https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#type

Reply:
@tunacicek and I adapted the URLs together